lua/cfc_disconnect_interface/client/cl_interface.lua

```diff
@@ -305,6 +312,11 @@ local function addButtonsBar( frame )
             barPanel.reconBtn:SetText( "YES" )
         else
             hideMessage()
+
+            timer.Simple( 0.25, function()
+                showMessage( "You'll have the option to respawn your props when you rejoin." )
+            end )
+
```

Comment:
`showMessage` does this delay stuff if there's already a message shown, the hide + delayed show here can be replaced with just `showMessage( "You'll have the option to respawn your props when you rejoin." )`